Gopkg.lock

```diff
@@ -508,7 +508,7 @@
     ".",
     "nl"
   ]
-  revision = "7c0b5944a3036fc8dc9f4cbf2f5c76dedd29af8b"
+  revision = "f504738125a57f35f87fc30fb69b8df75237ccde"
```

Comment:
Did you run dep ensure -update like this ? https://golang.github.io/dep/docs/daily-dep.html#updating-dependencies or did you change this file manually?

Reply:
Did it manually. But our travis CI test includes "dep ensure" that the link suggests. 
Thanks for attaching the link though!